config/locales/views/errors/en.yml

```
@@ -4,3 +4,4 @@ en:
     not_found:
       subtitle: You may have mistyped the address or the page may have moved.
       title: The page you were looking for doesn't exist.
+    session_expired: Your session has expired. Please log in again to continue.
```

You are not logged out. The form authenticity token has expired, so you need to resubmit the form. 
```suggestion
    session_expired: Your session has expired. Please try again.
```